pkg/git/git.go

```diff
@@ -40,14 +44,38 @@ func Add(path string) error {
 	if leafDir != "" {
 		path = leafDir + "/" + path
 	}
-
 	// Add all changes to the working directory
 	err = w.AddWithOptions(&git.AddOptions{
 		Path: path,
 	})
 	if err != nil {
 		return err
 	}
+	status, err :=w.Status()
+	if err !=nil{
+		return err
+	}
+	var fileName string
+	pd, _, err:=langdetect.FindProjectType()
+	if err!=nil{
+		return err
+	}
+	switch pd{
```

Comment:
Then that function will only return a string of file name, I thought it would mot be useful anywhere else or we can change a structure of `ProjectDetails` a bit to include the entry point file name in it!

Reply:
> Then that function will only return a string of file name

yep, the function can accept `pd` and return the file name. I think that's useful in langdetect to easily map things

Reply:
Ok